### ISSUE TITLE
vulcanize task fails silently after new elements added in ./app/elements

### ISSUE THREAD

**Issue by rowntreerob** [PolymerElements/polymer-starter-kit#225]
to post an issue that i had extending the project, adding my own elements having child 'js' folders with js there instead of in-line in the element.

When you take the project and make this kind of addition, you expect existing gulp:copy task to do its job and to prepare for following vulcanize step to just work with the new assets under ./app/elements.

What actually happens is that vulcanize fails silently because the new element gets copied over to ./dist/elements WITHOUT its children. Vulcanize will require that "my-newelement/js/mynewelement.js" exists over in the copy tasks dest folder.

Well , until a developer makes the following change to "copy" task, ( gulp.js#97 ) it all fails in a way where there is not any good log...

OLD gulp.js:

  var elements = gulp.src(['app/elements/**/*.html'])
    .pipe(gulp.dest('dist/elements'));

New gulp.js after added a new element:

  var elements = gulp.src(['app/elements/**/*.html', 'app/elements/**/*.js'])
    .pipe(gulp.dest('dist/elements'));

Im a newby in Polymer but, it took a lot of digging in order to find out that 'vulcanize' task actually needed the 'js' folder for the new element to be over in ./dist/elements/my-newelement/js

**Comment by Frikki** [comment]
This is also true for external CSS. Adding CSS files to the copy task, however, doesn’t completely solve the issue, because `@apply(--some-var)` declarations aren’t handled, i.e., they are ignored when running `gulp serve:dist`.

**Comment by gazal-k** [comment]
I have the same problem that @Frikki has. I think its a gulp-vulcanize issue rather than a polymer-starter-kit issue though, and different from the issue mentioned by @rowntreerob. Check https://github.com/Polymer/vulcanize/issues/229

**Comment by samccone** [comment]
**TLDR** we should have things just work with external js / css

thanks for the awesome debugging / thinking here :a: :heavy_plus_sign: 
I opened a PR to hopefully resolve this pain, let me know what you think!
